builds/build.go

```diff
@@ -0,0 +1,93 @@
+package build
+
+import (
+  "fmt"
+  "time"
+
+  "github.com/Shyp/go-circle"
+  "github.com/Shyp/go-git"
+)
+
+func inSlice(a string, list []string) bool {
+  for _, b := range list {
+    if b == a {
+        return true
+    }
+  }
+  return false
+}
+
+// GetBuilds gets the status of the 5 most recent Circle builds for a branch
+func GetBuilds(branch string) error {
+  // Different statuses Circle builds can have
+  green := []string{"fixed", "success"}
+  grey  := []string{"retried", "canceled", "not_run"}
+  red   := []string{"infrastructure_fail", "timedout", "failed", "no_tests"}
+  blue  := []string{"running"}
+  // purple := []string{"queued", "not_running", "scheduled"}
+
+  tip, err := git.Tip(branch)
+  _ = tip
+  // This throws if the branch doesn't exist
+  if err != nil {
+    return err
+  }
+
+  fmt.Println("\nFetching recent builds for", branch, "starting with most recent commit\n")
+  // Give CircleCI a little bit of time to start
```

Comment:
We did this with `wait` to add a little bit of latency between a git push & a Circle build. Not sure we need to sleep here, since we're fetching all recent builds.